### PR TITLE
User story 4

### PR DESCRIPTION
**Pull request by Lora92**

### CODE REVIEW

File: src/main/java/hr/inovatrend/petManager/Controller/UserController.java [hunk 1]
```diff
@@ -54,4 +54,15 @@ private String userEdit(@PathVariable("id") Long id, @ModelAttribute User user)
     }
 
 
+    @GetMapping("/delete/{id}")
```
Comment (FilipPandi) [inline src/main/java/hr/inovatrend/petManager/Controller/UserController.java]: Koristiti @DeleteMapping ( drugi način @RequestMapping(method = RequestMethod.DELETE) )

File: src/main/java/hr/inovatrend/petManager/Controller/UserController.java [hunk 2]
```diff
@@ -54,4 +54,15 @@ private String userEdit(@PathVariable("id") Long id, @ModelAttribute User user)
...
+    private String deleteUser(@PathVariable("id") Long id) {
+
+        User user = userService.getUser(id);
```
Comment (FilipPandi) [inline src/main/java/hr/inovatrend/petManager/Controller/UserController.java]: Zašto se dohvaća user? Koliko vidim delete metoda briše po Idu, a ne po objektu usera.

File: src/main/java/hr/inovatrend/petManager/Controller/UserController.java [hunk 3]
```diff
@@ -54,4 +54,15 @@ private String userEdit(@PathVariable("id") Long id, @ModelAttribute User user)
...
+
+        userService.deleteUser(id);
```
Comment (FilipPandi) [inline src/main/java/hr/inovatrend/petManager/Controller/UserController.java]: Preimenovati u deleteUserById.

File: src/main/java/hr/inovatrend/petManager/Service/Impl/UserServiceImpl.java [hunk 4]
```diff
@@ -21,4 +21,11 @@ public User createUser(User user) {
     public User getUser(Long id){
         return userRepository.findUserById(id);
     }
+
+    public void deleteUser(Long id){
```
Comment (FilipPandi) [inline src/main/java/hr/inovatrend/petManager/Service/Impl/UserServiceImpl.java]: Preimenovati u deleteUserById

File: src/main/java/hr/inovatrend/petManager/Service/UserService.java [hunk 5]
```diff
@@ -8,6 +8,8 @@ public interface UserService {
 
     User getUser(Long id);
 
+   void deleteUser(Long id);
```
Comment (FilipPandi) [inline src/main/java/hr/inovatrend/petManager/Service/UserService.java]: Isto preimenovati u deleteUserById

File: src/main/java/hr/inovatrend/petManager/Controller/UserController.java (hunk 1 above)
Comment (PhatDave) [reply src/main/java/hr/inovatrend/petManager/Controller/UserController.java]: a kako ce deletemapping preko thymeleaf-a?
zar nisu sve <a> reference get by default?

File: src/main/java/hr/inovatrend/petManager/Controller/UserController.java (hunk 2 above)
Comment (Lora92) [reply src/main/java/hr/inovatrend/petManager/Controller/UserController.java]: Popravljeno 
[053274d](https://github.com/Lora9299/Pet-Manager/pull/5/commits/053274db14f9e41aec28900ed09bc34ea28d7549)

File: src/main/java/hr/inovatrend/petManager/Controller/UserController.java (hunk 3 above)
Comment (Lora92) [reply src/main/java/hr/inovatrend/petManager/Controller/UserController.java]: Popravljeno 
[053274d](https://github.com/Lora9299/Pet-Manager/pull/5/commits/053274db14f9e41aec28900ed09bc34ea28d7549)

File: src/main/java/hr/inovatrend/petManager/Service/Impl/UserServiceImpl.java (hunk 4 above)
Comment (Lora92) [reply src/main/java/hr/inovatrend/petManager/Service/Impl/UserServiceImpl.java]: Popravljeno 
[053274d](https://github.com/Lora9299/Pet-Manager/pull/5/commits/053274db14f9e41aec28900ed09bc34ea28d7549)

File: src/main/java/hr/inovatrend/petManager/Service/UserService.java (hunk 5 above)
Comment (Lora92) [reply src/main/java/hr/inovatrend/petManager/Service/UserService.java]: Popravljeno 
[053274d](https://github.com/Lora9299/Pet-Manager/pull/5/commits/053274db14f9e41aec28900ed09bc34ea28d7549)

File: src/main/java/hr/inovatrend/petManager/Controller/UserController.java (hunk 1 above)
Comment (FilipPandi) [reply src/main/java/hr/inovatrend/petManager/Controller/UserController.java]: Ok, za sada nek' ostane tako, za ubuduće znaš da se treba koristiti delete mapping za delete (kod RESTa).

**Pull request merged**